### PR TITLE
Add regions which include states and provinces

### PR DESCRIPTION
**Pull request by joshuaRuff**
I added the "regions" keyword to each country. This allows one to see which States/Regions/Provinces a country has. This makes for great easy of use on front end forms that need access to global address.

### CODE REVIEW

File: data/countries.json [hunk 1]
```diff
@@ -2195,17 +5104,46 @@
...
+      "Kibuye",
+      "Kigali Rurale",
+      "Kigali-ville",
+      "Umutara",
+      "Ruhengeri"
     ]
   },
   "SA": {
-    "name": "Saudi Arabia",
+    "name": "v Arabia",
```
Comment (z-ax) [inline data/countries.json]: Looks like accidental change.

**Comment by z-ax** [comment]
Hi! I like the intent, but don't really like the implementation. One of the goals is to keep less traffic with this library on front-end, so all additional data should be generated separately. I'm thinking of something like `dist/{CODE}/regions.json` or something like this. Plus, our data has both English and Native translations of everything.

I know I've been a bit slow on maintaining this library recently. But now I'm thinking of the next major version to contain more data and re-organize it the way it's easy and clear to deploy and request for front-end. And hopefully will do this refactoring soon.

Planned regional data, cities and maybe geo-data to be able to generate maps — everything's available in different sources openly. Also thinking that maybe that should be a new package with a proper (more generic) name. Open for discussion.

**Comment by dmythro** [comment]
Hey @joshuaRuff, what was the source of data for this PR?

Working on #83, finally. Close to finish that. Now there's a room for additional data, like regions. Would be great to have this. But I think we need both English and Native region names. Maybe region codes (if any).

**Comment by dmythro** [comment]
Closing this PR, but feel free to contribute for v3 (my comment above).

**Pull request closed without merging**